etc/udiskie.profile

```diff
@@ -0,0 +1,35 @@
+# Firejail profile for udiskie
+# Description: Removable disk automounter using udisks
+# This file is overwritten after every install/update
+# quiet
+# Persistent local customizations
+include udiskie.local
+# Persistent global definitions
+include globals.local
+
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc include disable-programs.inc
+include disable-xdg.inc
+
```

Comment:
What about `include whitelist-var-common.inc`?

and `whitelist`ing in general?